src/Angor/Avalonia/AngorApp/UI/Shared/Services/UIServices.cs

```diff
@@ -22,6 +22,7 @@ public partial class UIServices : ReactiveObject
     [Reactive] private bool isDarkThemeEnabled;
     [Reactive] private bool isBitcoinPreferred = true;
     [Reactive] private bool isDebugModeEnabled;
+    private bool shouldSkipProductionValidations;
```

Comment:
`shouldSkipProductionValidations` is not the right name logically at this point, it was used at the flow in the point where we check validation then there is an if condition of "should we skip production validation" for the following properties.

the best name at this point is isDebugModeEnabled and to combine it with testnet so the property should be called `shouldSkipProductionValidations` -> `IsTestnetAndDebugModeEnabled`, that feel more appropriate. then in the validation we can have a property called `shouldSkipProductionValidations`